introduction.md

```diff
@@ -4,4 +4,4 @@
 * Javascript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?